Sources/Purchasing/StoreKitAbstractions/SK1StoreProductDiscount.swift

```diff
@@ -61,6 +61,14 @@ internal struct SK1StoreProductDiscount: StoreProductDiscountType {
 
 }
 
+#if swift(<5.7)
+// `SK1ProductDiscount` isn't `Sendable` until iOS 16.0 / Swift 5.7
+@available(iOS 11.2, macOS 10.13.2, tvOS 11.2, watchOS 6.2, *)
+extension SK1StoreProductDiscount: @unchecked Sendable {}
+#endif
```

Comment:
I wonder if we should keep all of these in one file. Then again, it's easy to look for `swift(<5.7)` in one pass whenever we're ready to remove the extra checks

Reply:
Yeah I can see an argument for both.